Build/macOS/build_macOS.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/sh
+
+#export LC_ALL=C.UTF-8
+export LANGUAGE=en_US.UTF-8
+export LC_ALL=en_US.UTF-8
+export LANG=en_US.UTF-8
+export LC_CTYPE=en_US.UTF-8
+
+set -e
+
+bin_folder="bin"
+version_prefix="1.1.10"
+dmg_name="Wasabi"
+echo "Enter apple is password:"
+read apple_id_password
```

Comment:
For security reasons we probably shouldn't echo back the password the user types. You can use the `-s` arg to not echo input coming from a terminal.

```suggestion
read -s apple_id_password
```